### PR TITLE
file-input and file-output

### PR DESCRIPTION
**Pull request by Andreto**
image
100%
image

### CODE REVIEW

**Comment by locness3** [comment]
why not

File: icons/file-output.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4"></path>
+  <path d="M14 2v6h6"></path>
+  <path d="M2 15l10-0"></path>
+  <path d="M5 12l-3 3 3 3"></path>
```
Comment (ericfennis) [inline icons/file-output.svg:15]:
```suggestion
  <path d="M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4" />
  <path d="M14 2v6h6" />
  <path d="M2 15l10-0" />
  <path d="M5 12l-3 3 3 3" />
```

File: icons/file-input.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4"></path>
+  <path d="M14 2v6h6"></path>
+  <path d="M2 15l10-0"></path>
+  <path d="M9 18l3-3-3-3"></path>
```
Comment (ericfennis) [inline icons/file-input.svg:15]:
```suggestion
  <path d="M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4" />
  <path d="M14 2v6h6" />
  <path d="M2 15l10-0" />
  <path d="M9 18l3-3-3-3" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Code formatting

**Pull request merged**